src/main/kotlin/com/terraformation/backend/support/atlassian/resource/CreateServiceDeskRequest.kt

```diff
@@ -0,0 +1,116 @@
+package com.terraformation.backend.support.atlassian.resource
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.terraformation.backend.config.TerrawareServerConfig
+import com.terraformation.backend.support.atlassian.SupportRequestType
+import io.ktor.client.call.body
+import io.ktor.client.request.HttpRequestBuilder
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.HttpResponse
+import io.ktor.http.HttpMethod
+import io.ktor.http.path
+import javax.inject.Named
+
+/**
+ * The POST request to create a service ticket.
+ *
+ * @param requestTypeId the ID of the JIRA issue type
+ * @param summary the title of the JIRA issue
+ * @param description the details of the JIRA issue
+ * @param reporter the email address of the reporter
+ */
+data class CreateServiceDeskRequest(
+    val description: String,
+    val summary: String,
+    val reporter: String? = null,
+    val requestTypeId: Int,
+    val serviceDeskId: Int
+) : AtlassianResource<PostServiceDeskRequestResponse> {
+  private val path = "/rest/servicedeskapi/request"
+  private val httpMethod = HttpMethod.Post
+
+  override fun buildRequest(requestBuilder: HttpRequestBuilder) {
+    val body =
+        PostServiceDeskRequestBody(
+            description = description,
+            requestTypeId = requestTypeId,
+            summary = summary,
+            raiseOnBehalfOf = reporter,
+            serviceDeskId = serviceDeskId,
+        )
+    with(requestBuilder) {
+      method = httpMethod
+      url { path(path) }
+      setBody(body)
+    }
+  }
+
+  override suspend fun parseResponse(response: HttpResponse): PostServiceDeskRequestResponse {
+    return response.body()
+  }
+
+  /** A builder class to inject configs or build information into the resource. */
+  @Named
```

Comment:
This will cause the class to be instantiated at application start and made available for dependency injection, which we don't want for a builder class; every caller is going to want a fresh new instance of the builder.